homepage/homepage.css

```diff
@@ -0,0 +1,146 @@
+:root{
+    --border-color:rgba(94, 91, 91, 0.493);
+    --footer-color:#242729
+}
+
+.span {
+    color: var(--main-Cta-color);
+}
+
+body {
+    height: 100%;
+}
+
+.search-bar input {
+    width: 15rem;
+    height: 2.5rem;
+    border-radius: var(--border-radius-5);
+    border: 2px solid var(--main-Cta-color);
+
+}
+
+.span-color {
+    color: var(--main-Cta-color)
+}
+
+.log-in,
+.nav-link-container a {
+    cursor: pointer;
+    font-weight:var(--font-bold);
+    text-decoration: none;
+    color: var(--text-secondry);
+
+}
+
+.log-in:hover,
+.nav-link-container a:hover {
+    color: var(--main-Cta-color);
+    border-bottom: 3px solid var(--main-Cta-color);
+}
+
+.hero-section {
+    display: flex;
+    flex-direction: row;
+    
+}
+
+.category {
+    text-align: center;
+
+}
+
+.category p {
+    font-size: var(--size-2);
+}
+
+.card-categories {
+    display: flex;
+    flex-direction: row;
+    gap: 3rem;
+    justify-content: center;
+    flex-wrap: wrap;
+    flex: 0 0 1;
+
```

Comment:
avoid empty lines before closing brace, where ever applicable in rest of the code